app/vep/models/client_model.py

```diff
@@ -1 +1,38 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
 
+class FieldOptions:
+    GENCODE_LABEL: str = "GENCODE"
+    GENCODE_VALUE: str = "gencode_comprehensive"
+
+class Options(BaseModel):
+  label: str = FieldOptions.GENCODE_LABEL
+  value: str = FieldOptions.GENCODE_VALUE
+
+class TranscriptSet(BaseModel):
```

Comment:
This should probably be something like a `SelectOption` class. There will be other options of this type in the future, I am sure.